IdentityCore/src/controllers/MSIDSilentController.m

```diff
@@ -121,28 +122,48 @@ - (void)acquireTokenWithRequest:(MSIDSilentTokenRequest *)request
             MSIDTelemetryAPIEvent *telemetryEvent = [self telemetryAPIEvent];
             [telemetryEvent setUserInformation:result.account];
             [telemetryEvent setIsExtendedLifeTimeToken:result.extendedLifeTimeToken ? MSID_TELEMETRY_VALUE_YES : MSID_TELEMETRY_VALUE_NO];
+            if (self.isLocalFallbackMode)
+            {
+                 [telemetryEvent setSsoExtFallBackFlow:1];
```

Comment:
Should this be excluded from MSAL CPP with a macro?

Reply:
It is excluded if you expanded the code a bit :)